src/components/views/room_settings/AliasSettings.js

```diff
@@ -160,12 +172,42 @@ module.exports = React.createClass({
         }
     },
 
+    addAlias: function(alias) {
+         // add this alias to the domain to aliases dict
+        var domain = alias.replace(/^.*?:/, '');
```

Comment:
@t3chguy domain was already declared, I just shifted it from onAliasAdded to addAlias function. So I din't make any changes in those declaration.:) The only problem I am facing here is when both the warnings are generated that is onChange and onAdd which is highly unlikely then the orientation of the onAdd warning gets disturbed that is do not fall in the tabel cell instead is seen pretty off from its actual position.